action.yaml

```diff
@@ -115,7 +118,8 @@ runs:
         TF_STATE_BUCKET: ${{ inputs.tf_state_bucket }}
         TF_STATE_BUCKET_DESTROY: ${{ inputs.tf_state_bucket_destroy }}
         REPO_ENV: ${{ inputs.repo_env }}
-        DOT_ENV: ${{ inputs.dot_env }}
+        GHS_ENV: ${{ inputs.dot_env }}
+        GHV_ENV: ${{ inputs.ghv_env }}
```

Comment:
This is what's happening ...
DOT_ENV got renamed to GHS for Github Secrets
GHV is Github variables